parser/src/main/java/com/microsoft/gctoolkit/parser/unified/UnifiedG1GCPatterns.java

```diff
@@ -60,11 +60,23 @@
 
     //[73.082s][info ][gc           ] GC(263) Concurrent Cycle
     //[2.179s][info ][gc          ] GC(9) Concurrent Mark Cycle
+//<<<<<<< HEAD
```

Comment:
Looks like you have an unresolved merge conflict. Or did you keep this around for historical reasons?